debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/PostgresConnectorIT.java

```diff
@@ -164,6 +164,16 @@ public void shouldValidateMinimalConfiguration() throws Exception {
                 configValue.errorMessages().isEmpty()));
     }
 
+    @Test
+    public void shouldNotValidatePgoutputDecoder() throws Exception {
+        Configuration config = TestHelper.defaultConfig()
```

Comment:
just curious, does the defaultConfig return a PG server version < 10?

Reply:
`Ig we can actually add an assert to make sure we are on version <10 for this test.` yeah let's do that as well in case the default behaviour changes for some reason, this assert should also be updated accordingly.

Reply:
Done